emscripten.py

```diff
@@ -1153,7 +1159,7 @@ def finalize_output(metadata, post, funcs_js, need_asyncify, provide_fround, asm
        access_quote('asmGlobalArg'), the_global,
        shared_array_buffer,
        access_quote('asmLibraryArg'), sending,
-       "'use asm';" if not metadata.get('hasInlineJS') and settings['ASM_JS'] == 1 else "'almost asm';", 
+       "'use asm';" if not metadata.get('hasInlineJS') and settings['ASM_JS'] == 1 and not settings['CYBERDWARF'] else "'almost asm';",
```

Comment:
I see. Yeah, that would not validate as asm.js, so it's correct to remove `use asm`.